### PR TITLE
New package: zutty-0.9

### PR DESCRIPTION
**Pull request by paxet**
Zutty is a GPU accelerated terminal emulator for the X Window System

#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

 
#### Does it build and run successfully? 
(Please choose at least one native build and, if supported, at least one cross build. More are better.)
- [x] I built this PR locally for my native architecture, (ARCH-LIBC)
- [x] I built this PR locally for these architectures (if supported. mark crossbuilds):
  - [x] aarch64-musl
  - [x] armv7l
  - [x] armv6l-musl

### CODE REVIEW

File: srcpkgs/zutty/template [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Template file for 'zutty'
+pkgname=zutty
+version=0.9
+revision=1
+build_style=waf3
+hostmakedepends="pkg-config"
+makedepends="python3-devel libglvnd-devel MesaLib-devel freetype-devel libXmu-devel"
+depends="libglvnd freetype libXmu font-misc-misc"
```
Comment (ericonr) [inline srcpkgs/zutty/template:8]: The first 3 should be detected automatically.

Comment (paxet) [reply srcpkgs/zutty/template:8]: What do you mean by the first 3?
It's my first contribution and don't know the ins and outs of package building.

Comment (ericonr) [reply srcpkgs/zutty/template:8]: The first three packages listed in `depends+`.

**Comment by paxet** [comment]
My bad, produced a merge branch.
Will reopen it

**Pull request closed without merging**